tests/integration/suite/test_system_project.py

```diff
@@ -71,3 +73,14 @@ def test_system_project_cant_be_deleted(admin_mc, admin_cc):
         admin_mc.client.delete(system_project)
     assert e.value.error.status == 405
     assert e.value.error.message == 'System Project cannot be deleted'
+
+
+def test_system_namespaces_default_svc_account(admin_mc):
+    k8sclient = CoreV1Api(admin_mc.k8s_client)
+    saccounts = k8sclient.list_service_account_for_all_namespaces()
+    for sa in saccounts.items:
+        if sa.metadata.name == "default":
+            sa_ns = sa.metadata.namespace
+            if sa_ns in initial_system_namespaces and sa_ns != "kube-system":
+                flag = sa.automount_service_account_token
+                assert flag is False
```

Comment:
To ensure the test isn't marked as a failure before the controller is done, use a wait_for on the check which will give the controllers time to complete their work.